phat/contracts/brick_profile/lib.rs

```diff
@@ -271,6 +289,14 @@ mod brick_profile {
             Ok(())
         }
 
+        /// Dump the secret key of a **dumped** EVM account, only owner is allowed
+        #[ink(message)]
+        pub fn dump_secret_key(&self, id: ExternalAccountId) -> Result<[u8; 32]> {
```

Comment:
Better to call "get_dumped_key" since this action is not dumping itself

Reply:
Renamed.